README.md

```diff
@@ -636,6 +629,19 @@ await appHistory.navigate(url, { replace: true, state, navigateInfo });
 
 Again, unlike `history.replaceState()`, `appHistory.navigate(url, { replace: true })` will by default perform a full navigation. And again, single-page apps will usually intercept these using `navigate`.
 
+Finally, we have `appHistory.reload()`. This can be used as a replacement for `location.reload()`, but it also allows passing `navigateInfo` and `state`, which are useful when a single-page app intercepts the reload using the `navigate` event:
+
+```js
+// Just like location.reload().
+await appHistory.reload();
+
+// Leave the state as-is, but pass some navigateInfo.
+await appHistory.reload({ navigateInfo });
+
+// Overwrite the state with a new value.
+await appHistory.reload({ state, navigateInfo });
```

Comment:
I don't instantly know how to implement this overwrite, but I assume it's solvable.